### PR TITLE
Feature/bend 004

### PR DESCRIPTION
**Pull request by OstapStrashevskii**
REST layer for Place Entity

### CODE REVIEW

**Review by SKuznet — COMMENTED** [review]
.gradle - should be at ignore

File: src/main/java/io/khasang/bend/entity/Place.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+@Entity
+@Table(name = "places")
+public class Place {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String name;
+    private String description;
+    private int album_id;//TODO with table albums??
```
Comment (SKuznet) [inline src/main/java/io/khasang/bend/entity/Place.java]: fix comments

**Review by SKuznet — CHANGES_REQUESTED** [review]

**Comment by OstapStrashevskii** [comment]
I have changed both 003 and 004 branches, commited and pushed this branches. What should I do then? Do I need restart pull request? or this new commits automatically changed in this pull request allready, and I have nothing to do else?

**Comment by OstapStrashevskii** [comment]
rm -r --cached .gradle from git

**Review by SKuznet — APPROVED** [review]

**Pull request merged**